daemon/mgr/container_utils.go

```diff
@@ -307,12 +317,19 @@ func validateResource(r *types.Resources) ([]string, error) {
 			warnings = append(warnings, warn)
 			r.CPUQuota = 0
 		}
+		// cpu.cfs_quota_us can accept value less than 0, we allow -1 and > 1000
+		if r.CPUQuota > 0 && r.CPUQuota < 1000 {
+			return warnings, fmt.Errorf("CPU cfs quota should be greater than 1ms(1000)")
+		}
 		if r.CPUPeriod > 0 && !cgroupInfo.CPU.CPUPeriod {
 			warn := "Current Kernel does not support cpu period, discard --cpu-period"
 			logrus.Warn(warn)
 			warnings = append(warnings, warn)
 			r.CPUPeriod = 0
 		}
+		if r.CPUPeriod != 0 && (r.CPUPeriod < 1000 || r.CPUPeriod > 1000000) {
```

Comment:
Since the number of `if` branch are too many, could we put the CPU related into one function? It can make the function readable....